javascript/ql/src/semmle/javascript/frameworks/Fastify.qll

```diff
@@ -132,7 +132,9 @@ module Fastify {
     string kind;
 
     RequestInputAccess() {
-      exists(string name | this.(DataFlow::PropRead).accesses(rh.getARequestExpr().flow(), name) |
+      exists(DataFlow::PropRead read, string name |
+        this = read and read = rh.getARequestSource().ref().getAPropertyRead(name)
```

Comment:
Is there a need for the `read` variable here?

Reply:
good spot.